net/DevExtreme.AspNet.Data.Tests/EFSortingTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using DevExtreme.AspNet.Data.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations {
+    class KeyAttribute : Attribute { }
+}
```

Comment:
It's an "ersatz" attribute used for testing - instead of referencing `System.ComponentModel.DataAnnotations`.